pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -600,6 +605,10 @@ func buildListenerFromEntry(builder *ListenerBuilder, le *outboundListenerEntry,
 				Name:       wellknown.HTTPConnectionManager,
 				ConfigType: &listener.Filter_TypedConfig{TypedConfig: protoconv.MessageToAny(hcm)},
 			}
+			opt.networkFilters = extension.PopAppendNetwork(opt.networkFilters, wasm, extensions.PluginPhase_AUTHN)
+			opt.networkFilters = extension.PopAppendNetwork(opt.networkFilters, wasm, extensions.PluginPhase_AUTHZ)
+			opt.networkFilters = extension.PopAppendNetwork(opt.networkFilters, wasm, extensions.PluginPhase_STATS)
+			opt.networkFilters = extension.PopAppendNetwork(opt.networkFilters, wasm, extensions.PluginPhase_UNSPECIFIED_PHASE)
```

Comment:
isn't it set in `buildCompleteNetworkFilters`

Reply:
Ah, this is  fro http, wondering could we do it in underlying like `buildCompleteNetworkFilters`

Reply:
I guess maybe in buildSidecarOutboundHTTPListenerOpts? not sure its much cleaner though.

Or were you thinking something else